### DISCUSSION TITLE
Minimal WebUI?

### DISCUSSION THREAD

**Discussion by abdeldjalil-fellah** [smallstep/certificates#2122] [General]
Is there any WebUI for step-ca to manage basic settings and view logs and issued certificates?

**Comment by hslatman** [comment]
Hi @abdeldjalil-fellah,

No, a web UI is not available as part of `step-ca`. For logs I suggest you capture them using a process manager and/or a logging solution. It is possible to read certificates from the database, but they'll need to be parsed from the storage format. 

Our commercial solution includes a web UI for managing multiple authorities and keeps track of issued certificates in a more user-friendly manner.
